scania-digital-design-system/tegel
# test(toast): added unit test cases

## **Describe pull-request**  
I added unit test cases for all variations of the Toast component + tests of all the most important props and slots.

## **Issue Linking:**  
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3046](https://tegel.atlassian.net/browse/CDEP-3046)    

## **How to test**  
1. Open branch on you local, run `npm run test` on root level
2. All test should be passing
3. GitHub test action should be passing too. 

## **Checklist before submission**
